utils/build-tooling-libs

```diff
@@ -271,6 +271,8 @@ class Builder(object):
             "-DEXPERIMENTAL_STRING_PROCESSING_SOURCE_DIR=" +
             os.path.join(SWIFT_SOURCE_ROOT,
                          "swift-experimental-string-processing"),
+            "-DSWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE=" +
+            os.path.join(SWIFT_SOURCE_ROOT, "swift-syntax"),
```

Comment:
Do we also need to deal with `SWIFT_PATH_TO_EARLYSWIFTSYNTAX_BUILD_DIR`?

Reply:
We aren’t actually building SwiftSyntax in `build-tooling-libs`, this is just necessary so we don’t hit this error

https://github.com/apple/swift/blob/e1ffd24932e13dd6c77855d7854893d1c323be22/CMakeLists.txt#L627-L629